values.yaml

```diff
@@ -268,11 +268,11 @@ prometheus-blackbox-exporter:
   # Configure resources
   resources:
     requests:
-      cpu: "5m"
-      memory: "7Mi"
+      cpu: "50m"
+      memory: "70Mi"
     limits:
-      cpu: "15m"
-      memory: "20Mi"
+      cpu: "100m"
+      memory: "200Mi"
```

Comment:
I found the functional test that checks if all prometheus targets are available was failing non deterministically because this component was crashing OOM.

Reply:
this change is not related to the rest of the pr